org.mwc.debrief.core/src/org/mwc/debrief/core/editors/PlotEditor.java

```diff
@@ -1643,14 +1645,32 @@ private boolean isVideoFile(final String fileName)
 
   private void openVideoPlayer(final String fileName)
   {
-    // CorePlugin.VIDEO_PLAYER_VIEW
     //#2940 #6
-    //now open a dialog box and get the start time
-    
+    //if we cannot get the start time from filename open the dialog
```

Comment:
@ayeshare - this is quite subjective, but it is it possible to keep this understanding of PlaentMayo formats inside the media_player plugin?   I guess an alternate showVideoPlayer that omitted the start-time value could try to parse it from the fileName parameter.

Ideally, I'd also like the VideoPlayerStartTimeDialog to remain private within the media_player plugin.